tests/idris2/basic054/input

```diff
@@ -0,0 +1,2 @@
+:exec main
+:q
```

Comment:
Note that this file is useless: `run` does not use an `input`.

Reply:
OK. Removed the file.